### PR TITLE
Adds 4.9.17 release notes

### PR DESCRIPTION
**Pull request by darshan-nagaraj**
No BZ.

To be merged when the erratas are shipped live. I will send a reminder.

OCP version: **Applicable only to 4.9 only**

Direct Doc Preview Link: https://deploy-preview-40924--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-9-release-notes.html#ocp-4-9-17

### CODE REVIEW

File: release_notes/ocp-4-9-release-notes.adoc [hunk 1]
```diff
@@ -2421,3 +2421,27 @@ link:https://access.redhat.com/solutions/6644371[{product-title} 4.9.15 containe
...
+
+* Previously, {product-title} Jenkins Sync Plugin did not synchronize config maps and image streams that have the label `role` set to `jenkins-agent`, intended to map into Jenkins Kubernetes plugin pod templates. Consequently, {product-title} Jenkins Sync Plugin no longer imported pod templates from the config maps or image streams with the `jenkins-agent` label. With this update, the accepted label specification is corrected. As a result, {product-title} Jenkins Sync Plugin imports pod templates from config maps or image streams with the `jenkins-agent` label. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2038961[*BZ#2038961*])
+
+
+[id="ocp-4-9-17-upgrading"]
+==== Upgrading
+
+To upgrade an existing {product-title} 4.9 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster within a minor version by using the CLI] for instructions.
```
Comment (kmccarron-rh) [inline release_notes/ocp-4-9-release-notes.adoc:2447]: s/upgrade/update/

**Review by kmccarron-rh — COMMENTED** [review]
A couple of small things - sending to @ahardin-rh   to give this a second set of peer review eyes.

File: release_notes/ocp-4-9-release-notes.adoc [hunk 2]
```diff
@@ -2421,3 +2421,27 @@ link:https://access.redhat.com/solutions/6644371[{product-title} 4.9.15 containe
...
+{product-title} release 4.9.17 is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2022:0195[RHBA-2022:0195] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2022:0194[RHBA-2022:0194] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6662631[{product-title} 4.9.17 container image list]
+
+[id="ocp-4-9-17-bug-fixes"]
+==== Bug fixes
+
+* Previously, csi-driver pod's `livenessProbe` had a low timeout. Consequently, the probe would fail on slower clouds causing the cluster to be degraded. With this update, timeout of `livenessProbe` is set to more realistic values to accommodate slower environments. As a result, the cluster is no longer degraded on clouds with slow cinder. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2037080[*BZ#2037080*])
```
[the remark this answers is not in the document]
Comment (darshan-nagaraj) [reply release_notes/ocp-4-9-release-notes.adoc:2439]: Thanks, I will do.

[the remark this answers is not in the document]
Comment (darshan-nagaraj) [reply release_notes/ocp-4-9-release-notes.adoc:2439]: @kmccarron-rh You can find it in BZ description https://bugzilla.redhat.com/show_bug.cgi?id=2037080#c0

File: release_notes/ocp-4-9-release-notes.adoc [hunk 3]
```diff
@@ -2421,3 +2421,27 @@ link:https://access.redhat.com/solutions/6644371[{product-title} 4.9.15 containe
...
+
+* Previously, {product-title} Jenkins Sync Plugin did not synchronize config maps and image streams that have the label `role` set to `jenkins-agent`, intended to map into Jenkins Kubernetes plugin pod templates. Consequently, {product-title} Jenkins Sync Plugin no longer imported pod templates from the config maps or image streams with the `jenkins-agent` label. With this update, the accepted label specification is corrected. As a result, {product-title} Jenkins Sync Plugin imports pod templates from config maps or image streams with the `jenkins-agent` label. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2038961[*BZ#2038961*])
+
+
+[id="ocp-4-9-17-upgrading"]
+==== Upgrading
```
[the remark this answers is not in the document]
Comment (darshan-nagaraj) [reply release_notes/ocp-4-9-release-notes.adoc:2445]: I will fix this. Thanks

File: release_notes/ocp-4-9-release-notes.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (darshan-nagaraj) [reply release_notes/ocp-4-9-release-notes.adoc:2447]: @kmccarron-rh I will fix this. Thanks

File: release_notes/ocp-4-9-release-notes.adoc [hunk 4]
```diff
@@ -2421,3 +2421,27 @@ link:https://access.redhat.com/solutions/6644371[{product-title} 4.9.15 containe
...
+
+[id="ocp-4-9-17-bug-fixes"]
+==== Bug fixes
+
+* Previously, csi-driver pod's `livenessProbe` had a low timeout. Consequently, the probe would fail on slower clouds causing the cluster to be degraded. With this update, timeout of `livenessProbe` is set to more realistic values to accommodate slower environments. As a result, the cluster is no longer degraded on clouds with slow cinder. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2037080[*BZ#2037080*])
+
+* Previously, {product-title} Jenkins Sync Plugin did not synchronize config maps and image streams that have the label `role` set to `jenkins-agent`, intended to map into Jenkins Kubernetes plugin pod templates. Consequently, {product-title} Jenkins Sync Plugin no longer imported pod templates from the config maps or image streams with the `jenkins-agent` label. With this update, the accepted label specification is corrected. As a result, {product-title} Jenkins Sync Plugin imports pod templates from config maps or image streams with the `jenkins-agent` label. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2038961[*BZ#2038961*])
```
[the remark this answers is not in the document]
Comment (darshan-nagaraj) [reply release_notes/ocp-4-9-release-notes.adoc:2441]: Sure. I will do. Thanks!

**Pull request merged**